pkg/interactive/interactive_client.go

```diff
@@ -587,15 +583,18 @@ func (c *InteractiveClient) queryCompleter(d prompt.Document) []prompt.Suggest {
 	if !c.isInitialised() {
 		return nil
 	}
+	if !cmdconfig.Viper().GetBool(constants.ArgAutoComplete) {
+		return nil
+	}
+
+	var s []prompt.Suggest
```

Comment:
Do we need this?

Reply:
Are you talking about s? ya it will be used in the return value